entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -9,99 +9,218 @@ uid: core/get-started/aspnetcore/new-db
   * **ASP.NET and web development** (under **Web & Cloud**)
   * **.NET Core cross-platform development** (under **Other Toolsets**)
 * [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core).
 
-## Create a new project in Visual Studio 2017
+# [.NET Core CLI](#tab/netcore-cli)
+
+* [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core).
+
+---
+
+## Create a new project
+
+# [Visual Studio](#tab/visual-studio)
 
 * Open Visual Studio 2017
 * **File > New > Project**
-* From the left menu select **Installed > Visual C# > .NET Core**.
+* From the left menu, select **Installed > Visual C# > .NET Core**.
 * Select **ASP.NET Core Web Application**.
 * Enter **EFGetStarted.AspNetCore.NewDb** for the name and click **OK**.
 * In the **New ASP.NET Core Web Application** dialog:
-  * Ensure the options **.NET Core** and **ASP.NET Core 2.1** are selected in the drop down lists
+  * Make sure that **.NET Core** and **ASP.NET Core 2.1** are selected in the drop-down lists
   * Select the **Web Application (Model-View-Controller)** project template
-  * Ensure that **Authentication** is set to **No Authentication**
+  * Make sure that **Authentication** is set to **No Authentication**
   * Click **OK**
 
-Warning: If you use **Individual User Accounts** instead of **None** for **Authentication** then an Entity Framework Core model will be added to your project in `Models\IdentityModel.cs`. Using the techniques you learn in this tutorial, you can choose to add a second model, or extend this existing model to contain your entity classes.
+Warning: If you use **Individual User Accounts** instead of **None** for **Authentication** then an Entity Framework Core model will be added to the project in `Models\IdentityModel.cs`. Using the techniques you learn in this tutorial, you can choose to add a second model, or extend this existing model to contain your entity classes.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* At a command prompt run the following command:
+
+   ```cli
+   dotnet new mvc -o EFGetStarted.AspNetCore.NewDb
+   ```
+
+---
 
 ## Install Entity Framework Core
 
-To install EF Core, you install the package for the EF Core database provider(s) you want to target. For a list of available providers see [Database Providers](../../providers/index.md). 
+To install EF Core, you install the package for the EF Core database provider(s) you want to target. For a list of available providers, see [Database Providers](../../providers/index.md). 
+
+# [Visual Studio](#tab/visual-studio)
 
 For this tutorial, you don't have to install a provider package because the tutorial uses SQL Server. The SQL Server provider package is included in the [Microsoft.AspnetCore.App metapackage](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?view=aspnetcore-2.1).
 
+# [.NET Core CLI](#tab/netcore-cli)
+
+For this tutorial, you use SQLite because it runs on all platforms that .NET Core supports.
```

Comment:
I like to avoid you except when it makes the sentence more clear.

For this tutorial, SQLite is used because it runs on all .NET Core supported platforms.